### PR TITLE
Bugfix for node cannot establish in k8s: Multi translate config address

### PR DESCRIPTION
**Pull request by leeyr338**
Bugfix for node cannot establish in k8s: Multi translate config address

### CODE REVIEW

**Comment by kaikai1024** [comment]
You can merge to your branch yourself.
And you don't need to clean the commits when finish it. `squash and merge` can be a choice after review

File: cita-network/src/node_manager.rs [hunk 1]
```diff
@@ -34,13 +34,15 @@ use std::{
     time::{Duration, Instant},
 };
 
-pub const DEFAULT_MAX_CONNECTS: usize = 4;
```
Comment (kaikai1024) [inline cita-network/src/node_manager.rs]: 666?

File: cita-network/src/node_manager.rs [hunk 2]
```diff
@@ -34,13 +34,15 @@ use std::{
...
+pub const DEFAULT_MAX_CONNECTS: usize = 666;
```
Comment (kaikai1024) [inline cita-network/src/node_manager.rs]: 666?

Comment (leeyr338) [reply cita-network/src/node_manager.rs]: I like 666.

**Comment by u2** [comment]
The feature/network-p2 branch is a private branch for yourself or a public branch for more users?

**Comment by kaikai1024** [comment]
> You can merge to your branch yourself.
> And you don't need to clean the commits when finish it. `squash and merge` can be a choice after review

Also need to clean the commits. you should treat your every commits seriously.

**Comment by kaikai1024** [comment]
> The feature/network-p2 branch is a private branch for yourself or a public branch for more users?
> 
> If it's just a private branch, pushing to your local fork repo is ok. Reorg the commits while PR.

we will discuss this.

File: cita-network/src/node_manager.rs (hunk 2 above)
Comment (kaikai1024) [reply cita-network/src/node_manager.rs]: ???? Just that? Is it ok for whatevre num?

**Pull request merged**